src/components/ContactCard.tsx

```diff
@@ -5,23 +5,31 @@ interface UserContactProps {
         nameSurname: string,
         mobile: number,
         position: string,
-        job: string,
-        favorite: boolean
+        jobTitle: string,
+        favorite: boolean,
+        avatar: string
     }
 }
 
 const ContactCard = ({ userContact }: UserContactProps) => {
     const favoriteIcon = "./images/favorite.svg";
-    const { nameSurname, position, job, mobile, favorite } = userContact;
+    const { nameSurname, position, jobTitle, mobile, favorite, avatar } = userContact;
     const mobileNum = mobile.toString();
+
+    const redirectToDetailsPage = (): void => {
+        console.log('here')
+        window.location.href = 'http://localhost:3000/aboutContact/sdf';
+    } 
+
     return (
-        <div className="contact-card-container">
-            <img src="./images/girl_2.jpg" className="avatar" alt="avatar" />
+        <div className="contact-card-container" onClick={()=>redirectToDetailsPage()}>
+            {/* <button onClick={()=>redirectToDetailsPage}>Click</button> */}
```

Comment:
I think you can remove this commented out block now

Reply:
removed commented code, thanks